CODE_OF_CONDUCT.md

```diff
@@ -60,7 +60,7 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-metriq@unitary.fund.
+metriq@unitary.foundation.
```

Comment:
I have heard from @beecastanon that this doesn't exist yet. Shall we create it? cc @WrathfulSpatula